pages/index.tsx

```diff
@@ -46,17 +46,17 @@ const Home: NextPage<cache> = ({ contributors }) => {
             </p>
 
             <Button
-              href="https://discord.gg/rsPNAxwweg"
+              href="https://docs.google.com/forms/d/e/1FAIpQLSfqaFTQsKdWxQINWxMgMp_eSy3b37GyiPJhUfKfH-B1Q90dug/viewform"
```

Comment:
Are you sure the end of this google form backlinks immediately to our Discord? It would be nice to say "Thank you for taking some time to answer some questions! We hve a Discord you can join."

Reply:
Thanks, I'm currently working on it. Regarding the survey backlinks to a page with a discord invite link, from what I've looked up, I need access to configure the google forms for that but I'm also not so sure if that's the only way to do that. For now, I've set it up so that when users click on the survey button, it opens in a new tab, and the current page redirects to the one with the discord link. Is that okay?